run_scans.py

```diff
@@ -87,7 +87,7 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
     pass
   elif tool == "fastLatency.py":
     scanType = "latency/trig"
-    dirPath = "%s/%s/%s/"%(dataPath,chamber_config[link],scanType)
+    dirPath = "%s/%s/%s/"%(dataPath,chamber,scanType)
```

Comment:
Sorry I'm confused, let me know where I am wrong... this is in the function launchArgs, defined beginning on line 6, and it looks to me that ```chamber``` is the fifth argument.

Reply:
mmhmm good point.  One wonders why we are using the `chamber` argument at all since it can be extracted from the `chamber_config` dictionary.

To reduce complexity can you remove the `chamber` argument?

Reply:
will do